woocommerce/payment-gateway/admin/class-sv-wc-payment-gateway-admin-order.php

```diff
@@ -0,0 +1,439 @@
+<?php
+/**
+ * WooCommerce Payment Gateway Framework
+ *
+ * This source file is subject to the GNU General Public License v3.0
+ * that is bundled with this package in the file license.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://www.gnu.org/licenses/gpl-3.0.html
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@skyverge.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade the plugin to newer
+ * versions in the future. If you wish to customize the plugin for your
+ * needs please refer to http://www.skyverge.com
+ *
+ * @package   SkyVerge/WooCommerce/Payment-Gateway/Admin
+ * @author    SkyVerge
+ * @copyright Copyright (c) 2013-2017, SkyVerge, Inc.
+ * @license   http://www.gnu.org/licenses/gpl-3.0.html GNU General Public License v3.0
+ */
+
+namespace SkyVerge\WooCommerce\PluginFramework\v5_0_0;
+
+defined( 'ABSPATH' ) or exit;
+
+if ( ! class_exists( '\\SkyVerge\\WooCommerce\\PluginFramework\\v5_0_0\\SV_WC_Payment_Gateway_Admin_Order' ) ) :
+
+/**
+ * Handle the admin order screens.
+ *
+ * @since 5.0.0-dev
+ */
+class SV_WC_Payment_Gateway_Admin_Order {
+
+
+	/** @var SV_WC_Payment_Gateway_Plugin the plugin instance **/
+	protected $plugin;
+
+
+	/**
+	 * Constructs the class.
+	 *
+	 * @since 5.0.0-dev
+	 *
+	 * @param SV_WC_Payment_Gateway_Plugin The plugin instance
+	 */
+	public function __construct( SV_WC_Payment_Gateway_Plugin $plugin ) {
+
+		$this->plugin = $plugin;
+
+		add_action( 'admin_enqueue_scripts', array( $this, 'enqueue_scripts' ) );
+
+		// capture feature
+		if ( $this->get_plugin()->supports( SV_WC_Payment_Gateway_Plugin::FEATURE_CAPTURE_CHARGE ) ) {
```

Comment:
might be nicer to add a method to the gateway plugin class like `supports_charge_capture` which would improve the readability here.